### PR TITLE
test(ui): unit tests for my data widget

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes:

- Added tests for MyDataWidget component
- Fix no data placeholder for mydata widget

Screenshot 2023-06-01 at 1 48 11 PM

#
### Type of change:

- [x] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/MyData/MyDataWidget/MyDataWidget.test.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  it('should not render view all for 0 length data', () => {
+    act(() => {
+      render(
+        <MemoryRouter>
+          <MyDataWidget />
+        </MemoryRouter>
+      );
+    });
+
+    expect(screen.queryByTestId('my-data')).not.toBeInTheDocument();
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/MyData/MyDataWidget/MyDataWidget.test.tsx:92]: Can we update the test id? something like `my-data-view-all`

Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/MyData/MyDataWidget/MyDataWidget.test.tsx:92]: Why?

Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/components/MyData/MyDataWidget/MyDataWidget.test.tsx]: Got it udpated

**Review by Sachin-chaurasiya — DISMISSED** [review]

**Review by ShaileshParmar11 — COMMENTED** [review]
one comment

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by shahsank3t — APPROVED** [review]

**Pull request merged**